### PR TITLE
bitwarden: add private-git-cloner

### PR DESCRIPTION
**Pull request by bbguimaraes**
/assign smarterclayton

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
@smarterclayton can we please determine if this meets your needs today @smarterclayton

File: ci-operator/populate-secrets-from-bitwarden.sh [hunk 1]
```diff
@@ -137,6 +137,11 @@ for login in "openshift-bot" "openshift-build-robot" "openshift-cherrypick-robot
 	oc label secret "github-credentials-${login}" "ci.openshift.io/managed=true"
 done
 
+# openshift-publish-robot also has a token that grants read-only
+# access to private repositories.
+update_secret generic "private-git-cloner" "$( format_field_value "openshift-publish-robot" private-git-cloner "oauth" )"
```
Comment (smarterclayton) [inline ci-operator/populate-secrets-from-bitwarden.sh]: I think this works, I may have to try it out manually.

Comment (smarterclayton) [reply ci-operator/populate-secrets-from-bitwarden.sh]: Looks like it works, will try it out in the real env once this merges.

**Comment by bbguimaraes** [comment]
Rebased, removed the `oc label` call (unnecessary after #3115, also removed the one added accidentally by #3104).

**Comment by stevekuznetsov** [comment]
/lgtm
woo

**Pull request merged**